### PR TITLE
Drop type specs for DQM/Integration python files

### PR DESCRIPTION
**Pull request by jfernan2**
#### PR description:

Drop type specs in  DQM/Integration/python configurations files, which drive the Online DQM clients at P5

Since with the change the mis-spelled variable name would give error, some "hidden" non-existing/wrong parameters or not actually declared in the original cloned module, have been identified in the process

Hence some extra DQM packages have been updated to make it work and some other parameters have been commented out since they do not actually exist in the plugin definition

#### PR validation:

Unit Tests are reproduced

### CODE REVIEW

File: DQM/Integration/python/clients/sistrip_dqm_sourceclient-live_cfg.py [hunk 1]
```diff
@@ -294,55 +294,55 @@
...
-    process.SiStripMonitorClusterReal.UseDCSFiltering = cms.bool(False)
+    process.SiStripMonitorDigi.UseDCSFiltering = False
+    process.SiStripMonitorClusterReal.UseDCSFiltering = False
 
     process.MonitorTrackResiduals_gentk.Tracks                 = 'initialStepTracksPreSplitting'
     process.MonitorTrackResiduals_gentk.trajectoryInput        = 'initialStepTracksPreSplitting'
-    process.MonitorTrackResiduals_gentk.TrackProducer          = cms.string('initialStepTracksPreSplitting')
-    process.TrackMon_gentk.TrackProducer    = cms.InputTag('initialStepTracksPreSplitting')
-    process.TrackMon_gentk.allTrackProducer = cms.InputTag('initialStepTracksPreSplitting')
+#    process.MonitorTrackResiduals_gentk.TrackProducer          = 'initialStepTracksPreSplitting' ##DOES NOT EXIST ON DQM/TrackerMonitorTrack/src/MonitorTrackResiduals.cc
```
Comment (jfernan2) [inline DQM/Integration/python/clients/sistrip_dqm_sourceclient-live_cfg.py:314]: @mtosi @mmusich if you could please confirm? Thanks!

File: DQM/Integration/python/clients/sistrip_dqm_sourceclient-live_cfg.py [hunk 2]
```diff
@@ -294,55 +294,55 @@
...
+    process.TrackMon_gentk.TrackProducer    = 'initialStepTracksPreSplitting'
+    process.TrackMon_gentk.allTrackProducer = 'initialStepTracksPreSplitting'
     process.SiStripMonitorTrack_gentk.TrackProducer = 'initialStepTracksPreSplitting'
 
     process.SiStripSources_TrkReco   = cms.Sequence(process.SiStripMonitorTrack_gentk*process.MonitorTrackResiduals_gentk*process.TrackMon_gentk)
 
     ### STRIP
     process.load("DQM.SiStripMonitorClient.SiStripClientConfigP5_cff")
-    process.SiStripAnalyser.UseGoodTracks  = cms.untracked.bool(True)
+#    process.SiStripAnalyser.UseGoodTracks  = True   #DOES NOT EXIST ON DQM/SiStripMonitorClient/plugins/SiStripAnalyser.cc
```
Comment (jfernan2) [inline DQM/Integration/python/clients/sistrip_dqm_sourceclient-live_cfg.py:323]: @mtosi @mmusich if you could please confirm? Thanks!

**Comment by jfernan2** [comment]
please test

File: DQM/Integration/python/clients/sistrip_dqm_sourceclient-live_cfg.py (hunk 2 above)
Comment (mmusich) [reply DQM/Integration/python/clients/sistrip_dqm_sourceclient-live_cfg.py:323]: the change *per se* looks OK (please remove the commented code as well), though the module is not cloned here while in the PR description there's written:
> ... "hidden" non-existing/wrong parameters or not actually declared in the original cloned module, have been identified in the process

by the way it seems there is a parameter with the same name configured here:

https://github.com/cms-sw/cmssw/blob/ac5ca816d5982e089eca9e7d0aec24c517922ced/DQM/SiStripMonitorClient/python/SiStripClientConfig_Tier0_cff.py#L11

and it doesn't seem to be registered in any source code:

https://cmssdt.cern.ch/dxr/CMSSW/search?q=UseGoodTracks&case=true&redirect=true

File: DQM/Integration/python/clients/sistrip_dqm_sourceclient-live_cfg.py (hunk 1 above)
Comment (mmusich) [reply DQM/Integration/python/clients/sistrip_dqm_sourceclient-live_cfg.py:314]: the change *per se* looks OK (please remove the commented code as well), though the module is not cloned here while in the PR description there's written:
> ... "hidden" non-existing/wrong parameters or not actually declared in the original cloned module, have been identified in the process

File: DQM/Integration/python/clients/sistrip_dqm_sourceclient-live_cfg.py (hunk 2 above)
Comment (jfernan2) [reply DQM/Integration/python/clients/sistrip_dqm_sourceclient-live_cfg.py:323]: Thanks!

**Comment by jfernan2** [comment]
please test

**Comment by mmusich** [comment]
@jfernan2 I was wondering if a general tool to compare the output of the online DQM clients exists (ad if yes it can be made part of the integration tests)

**Comment by jfernan2** [comment]
Thanks for the suggestion @mmusich !
The tool compareHistograms.py and compareDQMOutput.py exist in CMSSW and these are what I have used to compared manually the Online root files produced by this PR with baseline.
However, making this comparion part of the integration tests would imply changes in the DQM test GUI which hosts this kind of comparisons, since Online root files are not properly displayed due to ROOT folders, just RelVal or Offline ones...
With the transition to the new DQM GUI, this test may be easier to implement

**Comment by jfernan2** [comment]
+1

**Comment by qliphy** [comment]
+1

**Pull request merged**